tests/MongoDB.Driver.Tests/Linq/Translators/AggregateGroupTranslatorTests.cs

```diff
@@ -125,6 +121,16 @@ public void Should_translate_count()
             result.Value.Result.Should().Be(1);
         }
 
+        [Fact]
+        public void Should_translate_count_with_a_predicate()
```

Comment:
Let's also add a test that looks like this:

```
var result = Group(x => x.A, g => new { Result = g.Where(x => x.A != "Awesome").Count() });
```

They should translate identically.